src/core/render/q-render.unit.tsx

```diff
@@ -304,14 +304,18 @@ describe('q-render', () => {
         expectRendered(
           <hello-world
             on:q-render={HelloWorld.onRender}
-            q:style={HelloWorld.styles as any}
+            q:sstyle={HelloWorld_styles}
```

Comment:
is `q:sstyle` correct?

Reply:
Scoped Style. Not loving it, but for now until we figure out something better.